content/logs/index.md

```diff
@@ -130,15 +133,59 @@ logs:
 
 {{< /highlight >}}
 
+### Filter logs
+
+All logs are not equals and you might want to send only a specific subset of logs to Datadog.
+To achieve this use the `log_processing_rules` parameter in your configuration file with the **exclude_at_match** `type`
+
+If the pattern is contained in the message the log is filtered, and not sent to Datadog.
+
+Example: Filter logs with datadog users
```

Comment:
Filter out logs where the user field matches the datadoghq.com domain.

Reply:
👍